sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ApproximateUniqueTest.java

```diff
@@ -276,6 +301,50 @@ public void testApproximateUniqueWithDifferentSampleSizes() {
     }
   }
 
+  /**
+   * Test ApproximateUniqueCombineFn. TestPipeline does not use combiners.
+   */
+  @RunWith(Parameterized.class)
+  public static class ApproximateUniqueCombineFnTest {
```

Comment:
There's only 3 parameters - I'd prefer a private utility method and 3 test methods calling it with each of the 3 parameters...

Reply:
Done.